src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+$--main-color: #fff;
+$--code-color: #616070;
+$--button-color: #00acdc;
+$--text-color: #060b35;
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: $--main-color;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $--text-color;
+  }
+
+  &__code {
+    margin: 4px 84px 0 16px;
+    line-height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+    color: $--code-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 16px 0;
+
+    &-text {
+      margin: 0;
+      line-height: 18px;
+      font-size: 12px;
+      color: $--code-color;
+    }
+
+    &-amount {
+      margin: 0;
+      line-height: 18px;
+      font-weight: 700;
+      font-size: 16px;
+      color: $--text-color;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    margin: 16px;
+    align-items: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border-radius: 5px;
+    text-decoration: none;
+    background-color: $--button-color;
+    color: $--main-color;
+
+    &:hover {
+      background-color: $--main-color;
+      border: 1px solid $--button-color;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9, [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover

Reply:
Please fix the comment by @dffUqp